### DISCUSSION TITLE
SwitchPresetWindowHight, Command to cycle proportional hight similar to SwitchPresetColumnWidth but for the window hight.

### DISCUSSION THREAD

**Discussion by TheAngusMcFire** [YaLTeR/niri#653] [Ideas]
I find myself reaching to the mouse every time I want to resize a window within a column which contains multiple windows. 
I usually have 2 windows per column and the upper one is about 3 quarters the high of the monitor. 
I would like to implement a command similar to SwitchPresetColumnWidth but for quickly changing the height. 
I would like to add another config section which would contain the height proportions which the windows can have, and the command.
Is this something you might be interested in, or have some other ideas regarding a solution?

**Comment by YaLTeR** [comment]
I guess with the new height distribution algorithm this should be somewhat easier to implement since resizing one window always converts all other windows in the column to weighted automatic. So you could add `WindowHeight::Preset(idx)` like `ColumnWidth::Preset(idx)` and keep the rest as is, and only one window in a column will ever use that.

**Reply by TheAngusMcFire** [reply]
Sounds good, I give it a try :-)
